### PR TITLE
general toleration being applied

### PR DESCRIPTION
**Pull request by nickmoyni**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-5559

### Change description ###

Adding a generic toleration for testing, as adding to cve scanner, manager, and controller may not be possible with current  version of chart.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
looks wrong, see https://github.com/hmcts/shared-services-flux/blob/master/k8s/namespaces/neuvector/neuvector.yaml#L85

should be under `neuvector` assuming you're on the same chart version

**Comment by thomast1906** [comment]
Will give you a shout on slakc - https://github.com/neuvector/neuvector-helm/blob/master/charts/core/values.yaml#L29 

may be down to the chart version cft is using

**Comment by thomast1906** [comment]
https://github.com/hmcts/chart-neuvector/releases/tag/v1.2.6 CFT needs this

**Comment by timja** [comment]
> [hmcts/chart-neuvector@`v1.2.6` (release)](https://github.com/hmcts/chart-neuvector/releases/tag/v1.2.6) CFT may need this

if you have time you could switch back to upstream we sent them a PR a few months ago and they just merged it days ago,
https://github.com/neuvector/neuvector-helm/pull/115

https://github.com/hmcts/chart-neuvector/blob/master/neuvector-azure-keyvault/Chart.yaml#L12-L13

**Comment by nickmoyni** [comment]
tag 4.3.2 and chart 1.2.6 added, ,also repositioned code as per Tim request under neuvector.

File: apps/neuvector/neuvector/perftest/perftest.yaml [hunk 1]
```diff
@@ -29,10 +90,16 @@ spec:
...
     keyvault:
       name: cftapps-test
       resourcegroup: core-infra-test-rg
       subscriptionid: 8a07fdcd-6abd-48b3-ad88-ff737a4b9e3c
       tenantid: 531ff96d-0ae9-462a-8d2d-bec7c0b42082
       aadpodidbinding: neuvector
       licensesecretname: neuvector-license-dev
+    chart:
+      spec:
+        version: 1.2.6
```
Comment (thomast1906) [inline apps/neuvector/neuvector/perftest/perftest.yaml:105]: Lets see how this version goes, sds does use 1.2.5

**Review by thomast1906 — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**